docker/README.md

```diff
@@ -34,26 +34,28 @@ Docker images are currently available on Docker Hub [CPU](https://hub.docker.com
    ``` 
    Verify the models and configuration file is in this newly created directory. Also verify that the `mms_app_[cpu|gpu].conf` is updated.
 
-3. Run the downloaded Docker image
+3. Run the downloaded CPU container image
    ```bash
    docker run --name mms -p 80:8080 -itd -v /home/users/models/:/models awsdeeplearningteam/mms_cpu
    0c1862a2c30edd0f33391f7277b2c8bcba4a5f4cf22669f3b308b4078a5d3ce8
    ```
+   Note, to run GPU image refer [this section](#running-the-mms-gpu-docker)
```

Comment:
Suggested rephrase: "...refer to [this section]..."

Reply:
Done.